app/preprints/-components/submit/review/template.hbs

```diff
@@ -1,67 +1,117 @@
-                <img
-                    data-test-provider-image
-                    local-class='image'
-                    src={{this.providerLogo}}
-                    alt={{t 'preprints.submit.step-one.title'}}
-                />
-                <div local-class='text'>
-                    {{this.preprint.provider.name}}
+            <div local-class='content-container'>
+                <h4>
+                    {{t 'preprints.submit.step-five.preprint-title'}}
+                </h4>
+                <div local-class='display'>
+                    {{this.preprint.title}}
                 </div>
             </div>
+            <div local-class='content-container'>
+                <Preprints::-Components::PreprintAbstract @preprint={{this.preprint}} />
+            </div>
+            <hr>
         </div>
-        <div local-class='content-container'>
-            <div local-class='title'>
-                {{t 'preprints.submit.step-five.preprint-title'}}
+        <div local-class='step-container'>
+            <h3 local-class='title'
+                data-test-title-step-two
+            >
+                {{t 'preprints.submit.step-two.title'}}
+            </h3>
+            <div local-class='content-container'>
+                <div local-class='title'>
+                    {{t 'preprints.submit.step-five.contributors'}}
+                </div>
+                <div local-class='display'>
+                    <ContributorList
+                        @model={{this.preprint}}
+                        @shouldTruncate={{false}}
+                        @shouldLinkUsers={{true}}
+                    />
+                </div>
             </div>
-            <div local-class='display'>
-                {{this.preprint.title}}
+            <div local-class='content-container'>
+                <div local-class='display'>
+                    <Preprints::-Components::PreprintLicense @preprint={{this.preprint}} />
+                </div>
             </div>
-        </div>
-        <div local-class='content-container'>
-            <div local-class='title'>
-                {{t 'preprints.submit.step-five.description'}}
+            <div local-class='content-container'>
+                <div local-class='display'>
+                    <Preprints::-Components::PreprintDiscipline @subjects={{this.subjects}} />
+                </div>
             </div>
-            <div local-class='display'>
-                {{this.preprint.description}}
+            <div local-class='content-container'>
+                <div local-class='display'>
+                    <Preprints::-Components::PreprintTag @preprint={{this.preprint}} />
+                </div>
             </div>
+            <div local-class='content-container'>
+                <h4>
+                    {{t 'preprints.submit.step-five.publication-doi'}}
+                </h4>
+                <div local-class='display'>
+                    {{this.preprint.preprintDoiUrl}}
```

Comment:
Does this field display what's entered in the Metadata page? It looks like in the Metadata page, the [input is bound to `Preprint.publicationDoi`](https://github.com/CenterForOpenScience/ember-osf-web/blob/feature/preprints-phase-2/app/preprints/-components/submit/metadata/template.hbs#L98), and this [`Preprint.preprintDoiUrl` is a computed property](https://github.com/CenterForOpenScience/ember-osf-web/blob/feature/preprints-phase-2/app/models/preprint.ts#L105) that we shouldn't directly be setting.

I think the correct field for the input to be bound to would be `Preprint.doi`, and here, we would just show `{{this.preprint.doi}}` as well